app/Module/GoogleMapsModule.php

```diff
@@ -4692,8 +4704,10 @@ function initialize() {
 			//--------------------------------------------------------------------------------------
 			var street = new google.maps.ImageMapType({
 				getTileUrl: function(coord, zoom) {
-					var X = coord.x % (1 << zoom);  // wrap
-					return 'https://cbk0.google.com/cbk?output=overlay&zoom=' + zoom + '&x=' + X + '&y=' + coord.y + '&cb_client=api';
+					//var X = coord.x % (1 << zoom);  // wrap
+					//return 'https://cbk0.google.com/cbk?output=overlay&zoom=' + zoom + '&x=' + X + '&y=' + coord.y + '&cb_client=api';
+					//return "http://tile.openstreetmap.org/" + zoom + "/" + X + "/" + coord.y + ".png";
+					return '';
```

Comment:
I actually sent you an email a few weeks ago asking you about this. I can only find a few references to cbk0.google.com and they were about obtaining metadata - it seems that this url has completely gone now. I have no idea at all what it was doing and that is why I left it commented out just in case someone else could shed any light.